coalib/__init__.py

```diff
@@ -6,10 +6,60 @@
 
 
 import sys
+from os import chdir, getcwd
 from os.path import join, dirname
+from subprocess import check_output, CalledProcessError
 
 
-VERSION_FILE = join(dirname(__file__), 'VERSION')
+COALIB_DIR = join(dirname(__file__))
+
+VERSION_FILE = join(COALIB_DIR, 'VERSION')
+
+COMMIT_FILE = join(COALIB_DIR, 'COMMIT')
+
+
+def get_current_commit(git_dir):
+    '''
+    :param git_dir:     string containing the directory to check the
+                        current commit in.
+    :return:            string containing the current commit sha if `git_dir`
+                        is initialized as a git repo otherwise an empty string.
+    '''
+    old_dir = getcwd()
+    chdir(git_dir)
```

Comment:
if we are already in the git repo dir (or any subdirectory), this `chdir` and the one below are not necessary.